services/terraformer/templates/azure.tpl

```diff
@@ -197,10 +199,21 @@ resource "azurerm_virtual_machine_extension" "{{ $node.Name }}_{{ $clusterHash }
   protected_settings = <<PROT
   {
       "script": "${base64encode(<<EOF
+      # Allow ssh as root
       sudo sed -n 's/^.*ssh-rsa/ssh-rsa/p' /root/.ssh/authorized_keys > /root/.ssh/temp
       sudo cat /root/.ssh/temp > /root/.ssh/authorized_keys
       sudo rm /root/.ssh/temp
       sudo echo 'PermitRootLogin without-password' >> /etc/ssh/sshd_config && echo 'PubkeyAuthentication yes' >> /etc/ssh/sshd_config && echo "PubkeyAcceptedKeyTypes=+ssh-rsa" >> sshd_config && service sshd restart
+      
+      {{- if not $nodepool.IsControl }}
+      # Mount managed disk only when not mounted yet
+      if ! grep -qs "/dev/sdc" /proc/mounts; then
```

Comment:
same comment as in Hetzner tpl..with one difference - how do we know the disk gets recognised as `sdc` every time? this mapping is unreliable and should not be used.

`sdc` might be another drive next reboot and we should be getting it's UUID/PARTID instead...